src/tactic/ui/tools/custom_layout_edit_wdg.py

```diff
@@ -2585,6 +2586,7 @@ def build_xml(cls, view, html, style=None, behavior=None, mako=None, kwargs=None
         html = html.replace("<%", "<![CDATA[\n<%")
         html = html.replace("%>", "%>]]>")
         html = html.replace("&", "&amp;")
+        html = re.sub(r'(\<\w+\>)(\</\w+\>)',r'\1 \2', html, re.MULTILINE)
```

Comment:
OK done. I looked at [python re module](https://docs.python.org/2/library/re.html) and it seems that '<' and '>' is used only in conjunction with '?' . So this should be safe (if not better).